packages/dataviews/src/types.ts

```diff
@@ -40,40 +40,103 @@ export interface Field {
 	/**
 	 * The width of the field column.
 	 */
-	width?: string | number;
+	width: string | number | undefined;
```

Comment:
There's a subtle difference between the two notations:

 - Previous says width is optional 
 - Now, it says width is not optional but "undefined" is a valid value for it.

This impacts the use of `Required` to define the `NormalizedField`. In other words, `Required` won't have any impact on these properties. It will only override the "optional" ones.

Reply:
Do we really need this? Why should we distinguish Field and NormalizedField in the first place? 🤔

Reply:
We do need this. We want components to know that render is always defined for NormalizedField for instance. This will prevent components from needed to do `if ( field.render ) fallback` and rely on the fact that these properties are always defined.